stdlib/gc.pyi

```diff
@@ -1,12 +1,16 @@
 import sys
-from typing import Any
-
-DEBUG_COLLECTABLE: int
-DEBUG_LEAK: int
-DEBUG_SAVEALL: int
-DEBUG_STATS: int
-DEBUG_UNCOLLECTABLE: int
-callbacks: list[Any]
+from typing import Any, Callable
+from typing_extensions import Literal
+
+DEBUG_COLLECTABLE: Literal[2]
+DEBUG_LEAK: Literal[38]
+DEBUG_SAVEALL: Literal[32]
+DEBUG_STATS: Literal[1]
+DEBUG_UNCOLLECTABLE: Literal[4]
+
+_CallbackType = Callable[[Literal["start", "stop"], dict[str, int]], Any]
```

Comment:
(It seems, I forgot to click "Start a review" earlier. Sorry about that.)

We're using `object` now instead of `Any` for unused return types or callbacks:

```suggestion
_CallbackType = Callable[[Literal["start", "stop"], dict[str, int]], object]
```